roadmap.md

```diff
@@ -0,0 +1,60 @@
+# Usage of Epics
+An Epic is a grouping of smaller related issues/tasks that fits into a larger goal. An Epic can potentially span across multiple iterations. 
+
+To track how smaller issues fit into the larger goal, we use [task lists](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists).
+Breaking a large issue into smaller issues makes the work more manageable and enables team members to work in parallel. 
+It also leads to smaller pull requests, which are easier to review.
+
+>NOTE: To create an epic on Github, use the available Epic template.
+
+# Usage of Github issue template/forms
+We have [customized templates](https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/blob/main/.github/ISSUE_TEMPLATE/epic.md) that helps contributors in 
+creating new issues in the repository. When creating an issue, choose the template that best fits the issue you are about to create.  
+
+# Communicate
+- Use `@bf2fc6cc711aee1a0c2a/factorized-fleet` to alert the team about the issue. 
+- Use appropriate labels during the creation or the triaging of the issue. Labels help to categorise issues thus improving their discoverbility. Check out the existing labels on - https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/labels.  
+- Assign collaborators to issues to communicate responsibility.
+
+>NOTE: You can always reach out to the wider team via the available [communication channels](https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/wiki/How-to-communicate-in-the-project) 
+
+# Assigning issue to a Sprint
+
+A Sprint is defined as an [iteration field](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/best-practices-for-managing-projects#use-different-field-types). An issue is assigned to a Sprint during Sprint planning.
+
+
+# View the Roadmap
+The roadmap is defined in the [project view](https://github.com/orgs/bf2fc6cc711aee1a0c2a/projects/5) 
+The project view contains all the issues that have been triaged and potentially assigned to a Sprint.   
+
+The view is the source of truth and they are kept up to date to showing our roadmap.
+
+## How to add an issue to the Roadmap
+
+To add an issue to the Roadmap, go the [project](https://github.com/orgs/bf2fc6cc711aee1a0c2a/projects/5) and click on the `Add Item` button (displayed down the board). 
+Github will propose a search input, where you can search for issues and add them to the board.
+
+## How to assign an issue to the Sprint
```

Comment:
This is kind of duplicated with the heading 1 section above "Assigning issue to a Sprint". I would advise merging the two.